content/documentation/install-uninstall.md

```diff
@@ -248,133 +257,41 @@ If version number is returned, quire-cli was installed correctly. You can now le
+- Updating Quire will not effect your existing projects. It swill only take effect on any new projects you start after updating.
 {{< /q-class >}}
 
-## Install a New Theme
-
-TK
-
-## Uninstall Quire
-
-To uninstall Quire:
-
-1. From a new window in your command-line shell (you should be in your home/user directory where you initially installed Quire) type the following two commands:
-
-    ```tx
-    cd quire-cli
-    ```
-
-    ```tx
-    npm uninstall -g
-    ```
-
-2. In the folder/finder view, go to your user/home directory where you initially installed Quire, look for the `quire-cli` folder and delete it.
-
-## Troubleshooting
-
-**Downloading the Quire CLI to your computer from GitHub through the Terminal**
+To update Quire to the latest version, you can run the single install command again. In your command-line shell (Terminal on macOS and PowerShell in Administrator mode on Windows), type the following text and hit enter:
 
-If you have {{< q-def "two-factor authentication" >}} set-up, you may need to create a personal access token in GitHub to get Quire CLI to download properly. More information on creating a personal access token can be found [here](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token), or you can follow the instructions below.
-
-Follow these steps:
-
-1. Run
-
-    ```text
-    git clone https://github.com/gettypubs/quire-cli.git
-    ```
-
-2. For username: enter your Github Username
-
-   For Password: **follow the directions below.**
-
-3. In GitHub follow this link (https://github.com/settings/tokens) to access the personal token access token page. Or, while on Github.com, navigate to the upper-right corner of any page, click your profile photo, then click **Settings**. In the left sidebar, click **Developer settings**. In the left sidebar, click **Personal access tokens**.
-
-6. Click **generate new token**.
-
-7. Give your token a descriptive name.
-
-8. This page allows you to select the scopes or permissions you'd like to grant this token. In this instance, click **repo** at the very top.
-
-9. Scroll down to the bottom of the page and click **generate token**.
+```text
+npm install --global @thegetty/quire-cli
+```
 
-10. Copy the token to your clipboard before closing this window. For security reasons, after you navigate off the page, you will not be able to see the token again.(Treat your tokens like passwords, and keep them secret.)
+Note that while any *new* projects you start at this point will use the latest version of Quire just installed, older, previously started projects will remain untouched. This to avoid any unforeseen and unintended changes to publications that are completed or in progress as the version of Quire you use can effect layouts of a page, availablility of features, etcetera.
 
-**Resetting user permission for local directory**
+run commands from the new version of Quire on old Quire projects.
```

Comment:
I'm not sure what this line is? It's it a section header?

Reply:
just a mistake